pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -273,6 +289,16 @@ func RunStartBuild(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra
 	return exitErr
 }
 
+// isBuildLogTimeout returns true if the error indicates timeout getting build
+// logs (default is 10 seconds).
+func isBuildLogTimeout(err error) bool {
```

Comment:
`StatusError` seems pretty generic to me: https://github.com/kubernetes/kubernetes/blob/55f5e48047a1c3e38b917703153e5bf4e2dcb2e6/pkg/api/errors/errors.go#L42-L46